### ISSUE TITLE
Microsite Identifiers

### ISSUE THREAD

**Issue by igorkorenfeld** [18F/brand#270]
Create new PRs to update old site "anchor" to new "identifier" 

- [x] UX Guide (waiting on [PR review](https://github.com/18F/ux-guide/pull/265) 6/30)
- [x] Product Guide (waiting on [PR review](https://github.com/18F/product-guide/pull/166) 8/3)
- [x] Accessibility Guide (waiting on [PR review](https://github.com/18F/accessibility/pull/281) 8/3)
- [x] Content Guide ([PR merged](https://github.com/18F/content-guide/commit/66918afb684d2a048373b36fda434000d70556c6))
- [x] De-risking Guide ([PR merged](https://github.com/18F/derisking-gov-tech/commit/f803876ffa5ba2e0dd909b941ba7b55d01d52f4d))
- [x] Development Guide ([PR merged](https://github.com/18F/development-guide/commit/b75ab1f4504fde03f2f1350ddc915cd097ed2676))
- [x] Portfolios Site

**Comment by igorkorenfeld** [comment]
Opened a [PR](https://github.com/18F/ux-guide/pull/265) for the ux guide, can do similar or share with other guides

**Comment by afeld** [comment]
Attempt to add to the uswds-jekyll theme, which would allow us to solve the problem in a single place: https://github.com/18F/uswds-jekyll/pull/212

**Comment by afeld** [comment]
Would you be willing to link to the related pull requests up top, so it's easy to go back and see what changes were required for each?

**Comment by igorkorenfeld** [comment]
All for that idea but it seem like that PR was stalled and is now closed and that the theme is in maintenance mode. In which case, what would it take to move a new PR through? 

And no problem I'll put those links in just a bit.

**Comment by afeld** [comment]
uswds-jekyll is in maintenance mode because there hadn't been anyone available to make active improvements on it. If you're doing that work anyway (across the sites), we would _more than welcome_ the help on the theme, especially since that would cut out complexity/maintenance elsewhere.

> what would it take to move a new PR through?

Getting it merge-able! I think it was close.

**Comment by igorkorenfeld** [comment]
Awesome, that helps clarify. I'll take a look at replicating what I've done compared to the older PR and hopefully push something up.

**Comment by Dahianna** [comment]
Dahianna will follow up with teams to make sure they merge PR

**Comment by igorkorenfeld** [comment]
Footer / identifier [merged into USWDS-Jekyll ](https://github.com/18F/uswds-jekyll/pull/245)

**Comment by cannandev** [comment]
@igorkorenfeld Is this issue ok to close out? I think it was waiting on https://github.com/18F/ux-guide/pull/265 6/30), which has been merged.

**Comment by igorkorenfeld** [comment]
Yes, thank you!
